### DISCUSSION TITLE
Items deconstructable but without requireddeconstructor can be deconstructed at any deconstructor

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15375] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Alchemical table (a special deconstructor) is where you process (deconstruct) element materials
but items without `requireddeconstructor` can be deconstructed here too

discussed in #13922

> I highly recommend to add the following as a default value for <Deconstruct> components, if nothing is configured by the modder:
> requireddeconstructor="deconstructor"
> 
> This way if a modder for example just puts <Deconstruct time="5" />, it will automatically only become valid for the regular deconstructor and does not affect the item when put into a research station or other custom deconstructors unless it was specifically configured in the <Deconstruct> component. This would also solve the issue of any vanilla item getting deconstructed if put into a research station if that research station allows any item input.

discussed in #14656
although research station can now contain item with tag `researchmaterial` but you still need to override other vanilla item to make it can be put inside research station
for example: a new item can be researched along with alien circuitry but I still need to override alien circuitry to add tag "researchmaterial"

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in the unstable branch by making items default to `requireddeconstructor="deconstructor"`. The change will be included in the next update.
